frontend/constants.js

```diff
@@ -2,3 +2,4 @@
  * Current AppStudio DevSandbox K8s URL.
  */
 module.exports.K8sTargetURL = 'https://api-toolchain-host-operator.apps.appstudio-stage.x99m.p1.openshiftapps.com:443';
+module.exports.K8sTargetWS = 'wss://api-toolchain-host-operator.apps.appstudio-stage.x99m.p1.openshiftapps.com:443';
```

Comment:
Can you add something similiar to https://github.com/openshift/hac-core/blob/main/frontend/config/plugins.js#L26 and perhaps remove these two exports.

Reply:
Also, looks like this is not used anywhere